src/presentation/ui/page/databases.templ

```diff
@@ -0,0 +1,299 @@
+	componentForm "github.com/speedianet/os/src/presentation/ui/component/form"
+	componentStructural "github.com/speedianet/os/src/presentation/ui/component/structural"
+	presenterDto "github.com/speedianet/os/src/presentation/ui/presenter/dto"
+)
+
+script DatabasesIndexLocalState() {
+    document.addEventListener('alpine:init', () => {
+		Alpine.data('databases', () => ({
+			// Primary states
+			database: {},
+			databaseUser: {},
+			setDatabaseUserPassword(password) {
+				this.databaseUser.password = password;
+			},
+			resetPrimaryStates() {
+				this.database = {
+					'name': '',
+					'size': ''
+				}
+				this.databaseUser = {
+					'username': '',
+					'password': '',
+					'privileges': []
+				}
+			},
+			init() {
+				this.resetPrimaryStates();
+			},
+
+			// Auxiliary states
+			changeSelectedDatabaseType(databaseType) {
+				htmx.ajax(
+					'GET',
+					'/databases/?dbType=' + databaseType,
+					{
+						select: '#databases-content',
+						target: '#databases-content',
+						swap: 'outerHTML transition:true'
+					},
+				);
+			},
+			installDatabaseService(databaseType) {
+				htmx.ajax(
+					'POST',
+					'/api/v1/services/installables/',
+					{
+						swap: 'none',
+						values: { name: databaseType },
+					},
+				);
+			},
+			get shouldDisableCreateDatabaseSubmitButton() {
+				return this.database.name == ''
+			},
+			get shouldDisableCreateDatabaseUserSubmitButton() {
+				return this.database.name == '' || this.databaseUser.username == '' || this.databaseUser.password == ''
+			},
+
+			// Modal states
+			isCreateDatabaseModalOpen: false,
+			openCreateDatabaseModal() {
+				this.resetPrimaryStates();
+
+				this.isCreateDatabaseModalOpen = true;
+			},
+			closeCreateDatabaseModal() {
+				this.isCreateDatabaseModalOpen = false;
+			},
+			isDeleteDatabaseModalOpen: false,
+			openDeleteDatabaseModal(databaseName) {
+				this.resetPrimaryStates();
+
+				this.database.name = databaseName;
+				this.isDeleteDatabaseModalOpen = true;
+			},
+			closeDeleteDatabaseModal() {
+				this.isDeleteDatabaseModalOpen = false;
+			},
+			deleteDatabaseElement(databaseType) {
+				htmx.ajax(
+					'DELETE',
+					'/api/v1/database/' + databaseType + '/' + this.database.name + '/',
+					{swap: 'none'},
+				);
+				this.closeDeleteDatabaseModal();
+			},
+			isCreateDatabaseUserModalOpen: false,
+			openCreateDatabaseUserModal() {
+				this.resetPrimaryStates();
+
+				this.isCreateDatabaseUserModalOpen = true;
+			},
+			closeCreateDatabaseUserModal() {
+				this.isCreateDatabaseUserModalOpen = false;
+			},
+			createDatabaseUser(databaseType) {
+				htmx.ajax(
+					'POST',
+					'/api/v1/database/' + databaseType + '/' + this.database.name + '/user/',
+					{
+						swap: 'none',
+						values: {
+							username: this.databaseUser.username,
+							password: this.databaseUser.password,
+							privileges: this.databaseUser.privileges,
+						}
+					},
+				);
+				this.closeCreateDatabaseUserModal();
+			},
```

Comment:
Again, why isn't this on the modal form? That's quite strange, let's see.

Reply:
Yeah, I read the code, don't know why you choose to go with a button that calls an Alpine function to call HTMX. Why didn't you use a form like in every other occasion?

Reply:
Now it's useless. Removed!